recipes/dask-gateway/meta.yaml

```diff
@@ -0,0 +1,96 @@
+{% set name = "dask-gateway" %}
+{% set version = "0.2.0" %}
+
+package:
+  version: {{ version }}
+  name: {{ name|lower }}
+
+source:
+  url: https://github.com/jcrist/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: 1ae59cc611fb4a7dd1210bdf82e1eef151d450f144ae66d1d478582cd9f95964
+
+build:
+  number: 0
+  version: {{ version }}
+
+outputs:
+  - name: dask-gateway-base
```

Comment:
I might rename this to `dask-gateway` (and `dask-gateway-server-base` to `dask-gateway-server`), so names lineup with PyPI.

Reply:
done